src/resources/postcss/rsvp-v1.pcss

```diff
@@ -385,7 +385,7 @@
  */
 
 .tribe-block__rsvp {
-	font-family: Helvetica, sans-serif, arial;
+	font-family: var(--tec-font-family-sans-serif);
```

Comment:
this pcss file doesn't import the font variables

Reply:
They come from common-skeleton and common-full